gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -1168,16 +1168,17 @@ func (vm *VMKeeper) processStorageDeposit(ctx sdk.Context, caller crypto.Address
 					rlmPath, rlm.Deposit, ugnot.Denom, depositUnlocked, ugnot.Denom))
 			}
 
-			err := vm.refundStorageDeposit(ctx, caller, rlm, depositUnlocked, released)
+			err := vm.refundStorageDeposit(ctx, params.StorageFeeCollector, rlm, depositUnlocked, released)
 			if err != nil {
 				return err
 			}
 			d := std.Coin{Denom: ugnot.Denom, Amount: depositUnlocked}
 			evt := gnostd.StorageUnlockEvent{
 				// For unlock, BytesDelta is negative
-				BytesDelta: diff,
-				FeeRefund:  d,
-				PkgPath:    rlmPath,
+				BytesDelta:      diff,
+				FeeRefund:       d,
+				PkgPath:         rlmPath,
+				ReleaserAddress: caller,
```

Comment:
@zivkovicmilos @jefft0 
One opinion here about 
ReleaserAddress ?
Norman told my that we already have this address as part of the transaction, and is true so maybe we can just not include this field